feat: cleaner design of scheduled rewards

bajtos opened the pull request:
- refactor: consistent format of scheduled rewards
- fix: less space between the rewards and the unit
- feat: show only 6 decimal places in rewards

Originally, I wanted to use 3 decimal places. However, the screenshot from @patrickwoodhead showed a number `0.00031...`, which would be rendered as `0.0`. Therefore, we decided to use 6 decimal places instead.

I could not find any helper in ethers.js BigNumber class that would allow me to specify how many decimal places to show. I implemented the truncation myself using string manipulation.

### Before

<img width="654" alt="Screenshot 2023-11-09 at 10 12 21" src="https://github.com/filecoin-station/desktop/assets/1140553/087c06bd-7eb1-475e-9d9a-11e8caec5a16">

### After

<img width="606" alt="Screenshot 2023-11-09 at 10 12 37" src="https://github.com/filecoin-station/desktop/assets/1140553/d74a5967-3ada-4626-9fc7-a22a36574ea6">

bajtos commented:
Tests are failing because of an unrelated change, I am fixing them here: https://github.com/filecoin-station/desktop/pull/1093

patrickwoodhead commented:
Can we do 6 decimal places? I think that makes it more exciting to first time users.

Also, in your before and after, the number of zeroes has changed. Are you converting from FIL to attoFIL?

bajtos commented:
> Also, in your before and after, the number of zeroes has changed. Are you converting from FIL to attoFIL or something?

Good catch! I noticed this problem too and fixed it in [450f82d](https://github.com/filecoin-station/desktop/pull/1092/commits/450f82d56880876c61cabb7435da3f9fd1c0c20f)

I updated the PR description to mention 6 digits and show new screenshots.

bajtos commented:
@juliangruber hindsight please 🙏🏻

juliangruber commented on main/wallet.js:
```diff
@@ -84,7 +84,15 @@ function getBalance () {
+    .slice(0, 6)
+    // remove trailing zeroes
+    .replace(/0+$/, '0')
+  return [whole, truncated].join('.')
```
would be great to add unit tests for this, this kind of function lends itself perfectly (you want to check many edge cases)

bajtos replied:
Great suggestion! 😍 I should have thought about this myself. 🙈 

Here is the PR; the tests helped me to discover a bug.
https://github.com/filecoin-station/desktop/pull/1100
